### PR TITLE
simplify style lib exports

### PR DESCRIPTION
**Pull request by deomsj**
Simplify consumer css opt in:
 - from this: `import 'node_modules/@constructor-io/constructorio-ui-autocomplete/lib/mjs/components/Autocomplete/Autocomplete.css'`
  - to this: `import '@constructor-io/constructorio-ui-autocomplete/starter-styles.css';`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +59,6 @@
     "@types/react-dom": "^18.0.6",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
-    "copyfiles": "^2.4.1",
```
Comment (esezen) [inline package.json:61]: Love this

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "main": "lib/cjs/index.js",
   "module": "lib/mjs/index.js",
   "exports": {
+    "./starter-styles.css": "./lib/starter-styles.css",
```
[the remark this answers is not in the document]
Comment (deomsj) [reply package.json:8]: totally open to other names for this file!

May be over-thinking it, but motivation for `starter-styles.css` was to be explicit about intended use case for these styles.

What alternative names do you think would be more clear or simple here, @esezen?

Something like:
 - `styles.css` 
 - `default.css` 
 - `defaults.css`

[the remark this answers is not in the document]
Comment (esezen) [reply package.json:8]: I would prefer `styles.css` but again, I am not super opposed to any other name

**Review by esezen — APPROVED** [review]
LGTM! Thanks for the changes

**Pull request merged**